frontend/src/v5/ui/routes/dashboard/projects/containers/uploadFileForm/uploadList/uploadListItem/components/uploadListItemDestination/uploadListItemDestination.component.tsx

```diff
@@ -75,69 +76,94 @@ export const UploadListItemDestination = ({
 		}
 	}, []);
 
+	const errorMessage = errors.containerName?.message;
+
+	const onAutocompleteChange = (_, newValue: IContainer) => {
+		setValue(newValue || emptyOption);
+		onChange(newValue || emptyOption);
+		if (!newValue) {
+			setNewOrExisting('');
+			forceUpdate();
+		} else {
+			setNewOrExisting(!newValue._id ? 'new' : 'existing');
+		}
+		setDisableClearable(!newValue);
+	};
+
+	const onAutocompleteInputChange = (_, name: string) => {
+		if (name === value.name) onChange(value);
+	};
+
+	const getFilterOptions = (options: IContainer[], params) => {
+		let filtered: IContainer[] = filter(options, params);
+		const { inputValue } = params;
+
+		const isExisting = options.some((option: IContainer) => inputValue === option.name);
+		filtered = filtered.filter((x) => x.name !== value.name);
+		if (containersInUse.length === containers.length && !inputValue) {
+			filtered = [];
+		}
+		if (inputValue !== '' && !isExisting) {
+			filtered = [{
+				...emptyOption,
+				name: inputValue,
+			}, ...filtered];
+		}
+
+		return filtered;
+	};
+
+	const optionIsUsed = ({ name }: IContainer) => containersInUse.some((c) => c.name === name);
+
+	const getOptionDisabled = (option: IContainer) => !!option.name && optionIsUsed(option);
+
+	const getRenderOption = (optionProps, option: IContainer) => {
+		if (option._id) {
+			return (
+				<ExistingContainer
+					inUse={optionIsUsed(option)}
+					container={option}
+					latestRevision={option.latestRevision}
+					{...optionProps}
+				/>
+			);
+		}
+
+		if (errorMessage) return (<UnavailableContainer />);
```

Comment:
You need to include `{...optionProps}` with this since it contains the key. Otherwise the console complains about keys in lists

Reply:
Fixed